packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -112,23 +112,24 @@ import CommitIcon from '~icons/cy/commit_x14'
 import { IconTimeStopwatch } from '@cypress-design/vue-icon'
 import { gql } from '@urql/core'
 import { dayjs } from '../runs/utils/day.js'
+import { useI18n } from 'vue-i18n'
+
+const { t } = useI18n()
 
-// runNumber and commitHash dont currently exist in the query and therefore are being obtained as props instead
 gql`
 fragment DebugPage on CloudRun {
 	id
+  runNumber
 	createdAt
 	status
```

Comment:
nitpick, but somehow parts of this query got committed using tabs for indentation